main.js

```diff
@@ -29,9 +30,139 @@ function checkWindowSize() {
   if (window.innerWidth > 768) {
     mobileMenu.style.display = 'none';
     body.classList.remove('menu-open');
-    menu_bg.style.filter = '';
+    menuBg.style.filter = '';
     logo[0].style.filter = '';
   }
 }
 
-window.onresize = checkWindowSize;
\ No newline at end of file
+window.onresize = checkWindowSize;
+
+// Popup Window
+(function createProjects() {
+  const projectDetails = [
+    {
+      name: 'Facebook',
+      description: 'a simple clone of Facebook.com',
+      image: 'images/project_first.png',
+      technologies: ['HTML', 'CSS', 'JavaScript'],
+      live_link: 'https://marvin-nyalik.github.io/DestryMarkers/',
+      source_link: 'https://github.com/marvin-nyalik/DestryMarkers',
+    },
+    {
+      name: 'Dating',
+      description: 'connecting couples across borders',
+      image: 'images/project_second.png',
+      technologies: ['HTML', 'CSS', 'JavaScript'],
+      live_link: 'https://marvin-nyalik.github.io/DestryMarkers/',
+      source_link: 'https://github.com/marvin-nyalik/DestryMarkers',
+    },
+    {
+      name: 'Tonic',
+      description: 'national campaign against drug abuse',
+      image: 'images/project_third.png',
+      technologies: ['HTML', 'CSS', 'JavaScript'],
+      live_link: 'https://marvin-nyalik.github.io/DestryMarkers/',
+      source_link: 'https://github.com/marvin-nyalik/DestryMarkers',
+    },
+    {
+      name: 'Walker',
+      description: 'how to keep walking when you cannot keep walking',
+      image: 'images/project_fourth.png',
+      technologies: ['HTML', 'CSS', 'JavaScript'],
+      live_link: 'https://marvin-nyalik.github.io/DestryMarkers/',
+      source_link: 'https://github.com/marvin-nyalik/DestryMarkers',
+    },
+  ];
+  const projectContainer = document.getElementById('works');
+  projectDetails.forEach((project, index) => {
+    const articleTemplate = `
+      <article class="card">
```

Comment:
- [ ] Your portfolio design for the modal popup as shown below does not completely match the provided figma template, the image looks stretched on the desktop version. please klindly fix this design flaw, you can try doing this by using the css property `object-fit:cover` which makes the image look good within the viewport. :+1: 
- [ ] Also you are not making use of the appropriate fonts such as the font-weight and font-size for the technologies, please kindly look through the figma template to ensure you are correctly using the appropriate fonts as provided. :+1: 
- figma design
![des-f](https://github.com/marvin-nyalik/NyalikPortfolio/assets/89055502/ed95b93e-f25c-44d1-b9f4-a7c7809e9798)
- your design
![des](https://github.com/marvin-nyalik/NyalikPortfolio/assets/89055502/9ae443b7-633b-463a-b71e-fa09f1cd2a29)